src/Microsoft.Kusto.ServiceLayer/DataSource/IKustoClient.cs

```diff
@@ -22,14 +22,15 @@ public interface IKustoClient
         /// <summary>
         /// Executes a query or command against a kusto cluster and returns a sequence of result row instances.
         /// </summary>
-        Task<IEnumerable<T>> ExecuteControlCommandAsync<T>(string command, bool throwOnError, CancellationToken cancellationToken);
+        Task ExecuteControlCommandAsync(string command, bool throwOnError, CancellationToken cancellationToken);
```

Comment:
Changed Command to return a task instead of an IEnumerable because it shouldn't be returning data since it's not querying.